controllers/account/iam_test.go

```diff
@@ -91,7 +91,7 @@ func TestGetSTSCredentials(t *testing.T) {
 		nil, // no error
 	)
 
-	creds, err := getSTSCredentials(
+	creds, err := awsclient.GetSTSCredentials(
```

Comment:
I think it makes sense to move the tests to a new file `sts_assume_role_test.go`, so that they are next to what they are actually testing.

Reply:
I have no issue with that, but we would have to make the `[setupDefaultMocks](https://github.com/openshift/aws-account-operator/blob/master/controllers/account/account_controller_test.go#L54)` in the account_controller_test.go public. @iamkirkbater should I go ahead and make the change

Reply:
I wouldn't neccessarily make the function public but just instantiate the mocks directly. From what `setupDefaultMocks` returns, the only thing we're using in these tests is actually `mocks.mockCtrl`, which we can also just initialize with a single line. So, instead of making `setupDefaultMocks` public, id just change the first few lines of the Test Like so:
```
	mockCtrl := gomock.NewController(t)	
        nullLogger := testutils.NewTestLogger().Logger()
        mockAWSClient := mock.NewMockClient(mockCtrl)
	defer mockCtrl.Finish()

```

Reply:
I'm not able to call the mock pkg, from my test. I'm running into a "circular import dependency" error since the mock (pkg/awsclient/mock) is within the awsclient pkg where the sts_assume_role_test is located